lib/cisco_node_utils/yum.rb

```diff
@@ -20,8 +20,9 @@
 require_relative 'node_util'
 
 module Cisco
-  # This Yum class provides cisco package management functions through nxapi.
+  # This Yum class provides cisco package management functions.
   class Yum < NodeUtil
+    EXEC_IN_DEFAULT_NS = 'ip netns exec default sdr_instcmd'
```

Comment:
This is a bit misleading since `sdr_instcmd` is the actual command being executed - I would expect EXEC_IN_DEFAULT_NS to be simply `'ip netns exec default'`.